btk/create_observing_generator.py

```diff
@@ -34,8 +41,16 @@ def __iter__(self):
         return self
 
     def __next__(self):
-        observing_generator = []
-        for band in self.bands:
-            btk_survey = self.obs_conds(self.survey_name, band)
-            observing_generator.append(btk_survey)
+        if self.multiresolution:
+            observing_generator = {}
+            for s in self.survey_name:
+                observing_generator[s] = []
```

Comment:
it's a little cumbersome to have to remember the order of the bands, another option is to store the `band` and `survey_name` inside the `cutout` object.

Reply:
It can probably be done but I am not sure whether there is a real benefit ? The bands are in the same order as in the images. It would make sense to the band stored in the cutout too, but I am not sure if the related code will be more readable.

Reply:
You are right, I think this is fine as it is for now